<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgWndPickDeviceClass</class>
 <widget class="QDialog" name="dlgWndAskCmdClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>dlgWndPickDevice</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>380</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>

   <widget class="QRadioButton" name="radioDeviceCC">
    <property name="geometry">
      <rect>
        <x>25</x>
        <y>100</y>
        <width>430</width>
        <height>70</height>
      </rect>
    </property>
    <property name="text">
      <string>Citizen Card</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioDeviceCMD">
    <property name="geometry">
      <rect>
        <x>25</x>
        <y>180</y>
        <width>430</width>
        <height>70</height>
      </rect>
    </property>
    <property name="text">
      <string>Digital Mobile Key</string>
    </property>
   </widget>

   <widget class="QPushButton" name="btnCancel">
    <property name="geometry">
      <rect>
      <x>25</x>
      <y>325</y>
      <width>203</width>
      <height>30</height>
      </rect>
    </property>
    <property name="text">
      <string>&amp;Cancel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnOk">
    <property name="geometry">
      <rect>
      <x>253</x>
      <y>325</y>
      <width>202</width>
      <height>30</height>
      </rect>
    </property>
    <property name="text">
      <string>&amp;OK</string>
    </property>
    <property name="default">
      <bool>true</bool>
    </property>
    </widget>
   <widget class="QLabel" name="lblTitle">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>10</y>
      <width>361</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-image: url(:/Resources/alpha.png);</string>
    </property>
    <property name="text">
     <string>Enter you PIN</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblIcon">
    <property name="geometry">
     <rect>
      <x>415</x>
      <y>15</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QHBoxLayout"/>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnOk</sender>
   <signal>clicked()</signal>
   <receiver>dlgWndAskCmdClass</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>218</x>
     <y>290</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>213</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>dlgWndAskCmdClass</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>556</x>
     <y>364</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>237</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>